Documentation/BasicUsage.md

```diff
@@ -0,0 +1,238 @@
+# Basic Usage
+
+For starters we're going to import everything ready to use.  
+Most of the included functionality is similar to how you would deal with WebRTC in your browser.  
+We support most of the official WebRTC APIs, see this [document](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection) for more details.
+
+```javascript
+import {
+	ScreenCapturePickerView,
+	RTCPeerConnection,
+	RTCIceCandidate,
+	RTCSessionDescription,
+	RTCView,
+	MediaStream,
+	MediaStreamTrack,
+	mediaDevices,
+	registerGlobals
+} from 'react-native-webrtc';
+```
+
+## Registering Globals
+
+```javascript
+registerGlobals();
+```
+
+```javascript
```

Comment:
Maybe say that now you can use these APIs like in a browser.

Reply:
That was just going to be a list of what gets registered.
No worries.